index.html

```diff
@@ -1236,6 +1259,22 @@ <h2>Extract response headers</h2>
       </p>
 
       <ol class="algorithm">
+        <li>
+          <p>Let <var>url</var> be <var>request</var>'s URL.</p>
+        </li>
+        <li>
+          <p>Clear <var>url</var>'s <a>fragment</a>.</p>

```

Comment:
Maybe add a TODO or open an issue to rebase all this on Fetch and well defined concepts of what request's URL is?

Reply:
Definitely; "Network request" is not well defined in this spec, and this should really be linked to [RFC7232](https://httpwg.org/specs/rfc7230.html#section-2.1); However, Fetch is a much better fit for a W3C spec. Filed #166 for that.